athlinks-schema.ttl

```diff
@@ -174,4 +193,107 @@ athlinks:chipTime
 athlinks:agePercentage
   a owl:DataTypeProperty ;
   rdfs:domain athlinks:IntervalResult;
-  rdfs:range schema:Decimal .
+  rdfs:range xsd:decimal .
+
+athlinks:isHidden
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isVisible
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isEnabled
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isApproved
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isHandicap
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isNavigable
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:elevUnit
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:TerrainSpecifics;
+  rdfs:range xsd:float .
+
+athlinks:altitude
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:TerrainSpecifics;
+  rdfs:range xsd:float .
+
+athlinks:startMarked
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:finishMarked
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:distanceUnit
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:float .
+
+athlinks:integrity
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:int .
+
+athlinks:createUser
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:int .
```

Comment:
If thats that case, then I don't think we need this property. There are already properties that describe the same thing. One example is schema.og/creator. We should either use that or find another similar standard property that has a range of Person / Organization.